tests/lax_scipy_test.py

```diff
@@ -348,9 +349,14 @@ def scipy_fun(z, m=l_max, n=l_max):
           a_normalized[m, l] = c2 * a[m, l]
       return a_normalized
 
-    self._CheckAgainstNumpy(scipy_fun, lax_fun, args_maker,
-                            rtol=1e-5, atol=1e-5, check_dtypes=False)
-    self._CompileAndCheck(lax_fun, args_maker, rtol=1E-6, atol=1E-6)
+    with jax.debug_nans(True):
+      with jax.disable_jit(disable=disable_jit):
+        # dtype promotion needed because nonzero() fails otherwise.
```

Comment:
Can you say more about this? This sounds like a bug, and we should fix it rather than working around it in the test.